spec/tests/writer_spec.rb

```diff
@@ -52,7 +52,6 @@
   end
 
   it 'Should be able to write a Brick graph to a turtle file' do
-    @writer_brick.create_output
```

Comment:
Why were all the `create_output`s deleted?

Reply:
this may have been a misguided attempt to improve code readability but I just rolled create output into the write_output_to_file method. though maybe we don't want to generate those graphs every time we write